MREUnityRuntime/MREUnityRuntimeLib/App/IMixedRealityExtensionApp.cs

```diff
@@ -73,6 +74,11 @@ public interface IMixedRealityExtensionApp
         /// </summary>
         GameObject SceneRoot { get; set; }
 
+        /// <summary>
+        /// Current connection to the application server
+        /// </summary>
+        IConnection Connection { get; }
```

Comment:
I don't think we should be exposing this to the host app.  The connection is supposed to be form internal use within the runtime.